src/g728/g728float/CMakeLists.txt

```diff
@@ -15,9 +18,6 @@ add_test(g728-enc3-verify ${CMAKE_COMMAND} -E compare_files ../test_data/incw3.b
 add_test(g728-enc4 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/g728 -little enc ../test_data/in4.bin ../test_data/in4.bin.float.out)
 add_test(g728-enc4-verify ${CMAKE_COMMAND} -E compare_files ../test_data/incw4.bin ../test_data/in4.bin.float.out)
 
-add_test(g728-enc5 ${CMAKE_RUNTIME_OUTPUT_DIRECTORY}/g728 -little enc ../test_data/in5.bin ../test_data/in5.bin.float.out)
```

Comment:
Should we then also remove the test file?

Reply:
../test_data/in5.bin is used as test vector for the fixed point version so it should not be removed.

Reply:
Could do... I do not know why this reference was missing originally. 
I will generate that file and update the pull request.